model/countries/FR/FR-model.yaml

```diff
@@ -9,7 +9,6 @@ cut-off-tokens:
   - admin-area2
   - admin-area3
   - admin-area4
-  - locality2
```

Comment:
I think that it would be useful to update FR-descriptions.yaml with a description of what locality2 means (see model/countries/BR/BR-descriptions.yaml)